docs/guides/integration-mobile-overview.md

```diff
@@ -58,6 +49,108 @@ For details, refer to one of the following guides:
+
+### Managing the UID2 Token
+
+The following table summarizes the mobile integration options for managing the UID2 token, with links to the documentation for each option.
+
+| UID2 Mobile Implementation Option | Documentation Link |
+| :--- | :---|
+| Prebid.js | Either of the following implementation guides:<ul><li>[UID2 Client-Side Integration Guide for Prebid.js](../guides/integration-prebid-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Prebid.js](../guides/integration-prebid-client-server.md)</li></ul> |
+| UID2 Android SDK | SDK doc: [SDK for Android Reference Guide](../sdks/sdk-ref-android.md)<br/>Implementation Guide:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+| UID2 iOS SDK | SDK doc: [SDK for iOS Reference Guide](../sdks/sdk-ref-ios.md)<br/>Implementation Guide:<ul><li>[UID2 Client-Side Integration Guide for Mobile](../guides/integration-mobile-client-side.md)</li><li>[UID2 Client-Server Integration Guide for Mobile](../guides/integration-mobile-client-server.md)</li></ul> |
+
+[**GWH_SW01 on the above: is the standard Prebid.js implementation guide a correct link? Or should we just link to the optional Prebid.js section in the mobile implementation guide?**]
+
+### Sending the Token to the Bidstream
+
+There are many options for sending your UID2 tokens to the bidstream.
+
+The following table summarizes the options supported by UID2.
+
+| Scenario | Option | Integration Guide |
+| :--- | :--- | :--- |
+| You want to retrieve ads on Google | Secure Signals | [Google Ad Manager Secure Signals Integration Guide](integration-google-ss.md) |
+| You want to use Google GMA to integrate video, banner, interstitial, or native ads into your app | xxx | [UID2 GMA Plugin for Android Integration Guide](../guides/mobile-plugin-gma-android.md) or [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-gma-ios.md) |
+| You want to use Google IMA to integrate multimedia ads into your app | xxx | [UID2 GMA Plugin for Android Integration Guide](../guides/mobile-plugin-ima-android.md) or [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-ima-ios.md) |
+
+[**GWH_SW02 on the above: do they need to install our Secure Signals option and then GMA/IMA/both? Or are these three equal options?**]
+
+### Functionality Summary
+
+The following table summarizes the functionality available with the various integration options.
+
+| Implementation Option... | Client-Side or Server-Side | Generate UID2 Token from DII | Refresh UID2 Token | Supports Android | Supports iOS | Send Token to Bidstream |
```

Comment:
i think the options are:
1. UID2 SDK    
1. Google GMA
1. Google IMA 
1. Prebid Mobile SDK 

Don't need to mention Android/iOS but we can keep a single _Support Android/iOS_ columns 

_Send Token to Bidstream_ - technically all do but in various ways so maybe we could add these into the tables: 
1. UID2 SDK - should say "can retrieve token manually and use it wherever publishers see fit". Just wanna say the token can be used by publisher however they like but they have to manually retrieve and send it to somewhere.
2. Google GMA/IMA - Once setup, will be collected by Google GMA/IMA SDK automatically
3. Prebid Mobile - Once setup, will be added into the ad request to Prebid Server

We should also have a column to emphasise that the Prebid/GMA/IMA Plugins require the UID2 SDK to work ("Requires UID2 SDK for Android/iOS")

Reply:
maybe we should split the table into 2 -

1. The functionality summary for just UID2 SDK for Android/iOS
2. The functionality summary for Supplementary Enhancement for specific technologies (i dunno what's the best term here) to mention the Google GMA/IMA and prebid mobile options

Reply:
@@sunnywu:
1. Android and iOS are different SDKs so I don't want to lump them under UID2 SDK which is too general.
2. Any input re sequencing? Curious what we recommend/what's most popular. Prebid at top or bottom basically.
3. Made a combined column for Android/iOS support.
4. "UID2 SDK - should say "can retrieve token manually and use it wherever publishers see fit". Just wanna say the token can be used by publisher however they like but they have to manually retrieve and send it to somewhere." -- the table is about what our specific product does. The SDK doesn't send the token to the bidstream. On this column, I did it all with asterisks to explanations below.
5. Good point but instead of adding a column I included it with the title.
I imagine a more glamorous visual rendering is possible but let's at least make sure all is accurate for a starting point. Thanks a million. Lot of comments on this table which is a key one!